ndmg/scripts/ndmg_bids.py

```diff
@@ -148,7 +115,7 @@ def main():
     )
     parser.add_argument(
         "output_dir",
-        help="""The directory where the output
+        help="""The local directory where the output
         files should be stored. If you are running group
         level analysis this folder should be prepopulated
         with the results of the participant level analysis.""",
```

Comment:
while we're add it, we should probably remove the shit that talks about group level analysis, since that's not a thing

Reply:
update: I just did this with myself since it takes 2 seconds, then merged it in